### ISSUE TITLE
Conceptual guides > Component architecture needs to be completed

### ISSUE THREAD

**Issue by stevepiercy** [plone/documentation#1775]
## Links of affected pages in Plone Documentation, if any.

- https://6.docs.plone.org/conceptual-guides/components.html
- https://6.docs.plone.org/backend/subscribers.html (this one should go away)

The documentation source of `plone.app.dexterity` is at the root of this repo and has already been converted to MyST. It just needs someone with technical knowledge to port relevant bits and pieces.

https://github.com/plone/documentation/tree/6.0/plone.app.dexterity

## Description

The conceptual guide, Components, needs some love and attention. Specifically:

### Verify how to register a utility

- source docs?

### Adapters

- source docs?

### Lookup

- source docs?

### Events and Subscribers

- https://github.com/plone/documentation/issues/133
- https://github.com/plone/documentation/issues/753
- https://github.com/plone/documentation/issues/871
- https://github.com/plone/documentation/issues/931
- https://5.docs.plone.org/external/plone.app.dexterity/docs/advanced/event-handlers.html
- https://5.docs.plone.org/develop/addons/components/events.html

**Comment by Manik-Khajuria-5** [comment]
Is this issue still open? I’d be happy to help with it.

**Comment by stevepiercy** [comment]
@Manik-Khajuria-5 yup, this is still open. It's a difficult one, as are all the issues of missing documentation for the Plone 6 backend.

**Comment by Manik-Khajuria-5** [comment]
Okay I’ll try to implement this . 
Working on this issue

**Comment by Manik-Khajuria-5** [comment]
Hey @stevepiercy , I have completed the required documentation. Could you please help me understand how I can verify when it is constructed? Also, a local component is either a persistent object in the ZODB or is constructed dynamically.

**Comment by stevepiercy** [comment]
@Manik-Khajuria-5 sorry, I don't have any idea. Someone with that knowledge of the Plone backend will need to step up.

**Comment by Manik-Khajuria-5** [comment]
@stevepiercy okay got it I have completed the work on my end rest . now it depends on someone who have expertise in backend to add extra docs or If I somehow figured out what is happening in backend will try to add in docs Thanks.
